pom.xml

```diff
@@ -431,5 +435,33 @@
         <javaVersion>17</javaVersion>
       </properties>
     </profile>
+
+    <profile>
+      <id>celeborn-0.5</id>
```

Comment:
It seems that using this profile will fail to compile. Do we have any plans to support it? Or only the latest version is supported
```
BlazeCelebornShuffleReader.scala:65: one more argument than can be applied to constructor CelebornShuffleReader: (handle: org.apache.spark.shuffle.celeborn.CelebornShuffleHandle[K, _, C], startPartition: Int, endPartition: Int, startMapIndex: Int, endMapIndex: Int, context: org.apache.spark.TaskContext, conf: org.apache.celeborn.common.CelebornConf, metrics: org.apache.spark.shuffle.ShuffleReadMetricsReporter, shuffleIdTracker: org.apache.spark.shuffle.celeborn.ExecutorShuffleIdTracker)org.apache.spark.shuffle.celeborn.CelebornShuffleReader[K,C]
```

Reply:
https://github.com/kwai/blaze/pull/1144

fixed in this pr, also added GA